### ISSUE TITLE
My project is aspnet-webapi, the project needs upgrading to.Net Core, but you only support net47, net46 and net45, can.Net Core Support?

### ISSUE THREAD

**Issue by ghd258** [unitycontainer/aspnet-webapi#11]
My project is aspnet-webapi, the project needs upgrading to.Net Core, but you only support net47, net46 and net45, can.Net Core Support?

**Comment by ENikS** [comment]
@Chavoshi Would you be interested to help with this one as well?

**Comment by Chavoshi** [comment]
Hi @ENikS,
It's my pleasure to help.

**Comment by ghd258** [comment]
https://github.com/unitycontainer/aspnet-mvc  too net core support

**Comment by ghd258** [comment]
@ENikS https://github.com/unitycontainer/aspnet-mvc too net core support  Would you be interested to help with this one as well?

**Comment by ghd258** [comment]
@Chavoshi @ENikS   When is this a good problem?

**Comment by ENikS** [comment]
Do you want to try to do it yourself and send a PR?

**Comment by Chavoshi** [comment]
I tried to do the changes, but really it's not a simple job as DependencyResolver deprecated in ASP.NET Core and we need to implement them as ServiceProvider. I researched on this concept as well but it is not something overnight which we can put them in, as personally still there are a lot of blurry parts for me.

So in conclusion, in my opinion, this change is not a simple conversion and it needs to think more deeply and in addition it might need some infrastructural changes in core as well.
